src/IssueViz.Security/Models/IHotspot.cs

```diff
@@ -28,21 +28,14 @@ internal interface IHotspot : IAnalysisIssueBase
     {
         string HotspotKey { get; }
 
-        HotspotPriority Priority { get; }
+        IHotspotRule Rule { get; }
 
         /// <summary>
         /// File path as received from SQ server
         /// </summary>
         string ServerFilePath { get; }
     }
 
-    public enum HotspotPriority
-    {
-        High,
-        Medium,
-        Low
-    }
-
```

Comment:
Moved to the `IHotspotRule` file.

Reply:
@duncanp-sonar note that this field is also used in the xaml and I don't see a fix for it.
Does the colour of the priority column still work?

Reply:
It does now ;-) Good catch.